### PR TITLE
Fix axis in Softmax activation layer

### PR DESCRIPTION
**Pull request by janhuenermann**
Hey guys,

just stumbled across a tiny little issue in the Softmax activation layer and fixed it.

Cheers.
-Jan

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-layers/177)

### CODE REVIEW

**Comment by caisq** [comment]
:lgtm_strong:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-layers/177#-:-LCTyFYi-HDY4zwFvxQx:b-jbsf8z)*

**Comment by caisq** [comment]
Thanks, @janhuenermann !
